doc/tips-and-tricks-using-energyplus/src/utilities.tex

```diff
@@ -1 +1,3 @@
 \chapter{Utilities}\label{utilities}
+
+EnergyPlus comes with a wide variety of pre- and post-processing that are found in the PreProcess and PostProcess folders in the main EnergyPlus install folder.  These utilities have been developed since the initial release of EnergyPlus and can be helpful during the creation of input data for EnergyPlus or interpreting its output.
```

Comment:
Yeah that'll be better than nothing...literally!

Reply:
I know! 😂😂😂 How long has this been like this?🤦‍♂️ Thanks for reviewing this!